Java/API/src/edu/uml/cs/isense/api/API.java

```diff
@@ -450,6 +447,76 @@ public RDataSet getDataSet(int dataSetId) {
 		return result;
 	}
 
+ 	/**
+	 * Gets all the data sets associated with a project The data sets returned
+	 * by this function DO have their data field filled.
+	 *
+	 * @param projectId
+	 *            The project ID whose data sets you want
+	 * @return An ArrayList of Data Set objects
+	 */
+	public ArrayList<RDataSet> getFilledDataSets(int projectId) {
```

Comment:
This function is not lined up correctly. The first half of the function is indented way more than the second half. Check your editor's settings.